apps/antalmanac/src/components/RightPane/AddedCourses/AddedCoursePane.tsx

```diff
@@ -1,277 +1,322 @@
-                            color="inherit"
-                            onClick={() => {
-                                if (
-                                    window.confirm(
-                                        'Are you sure you want to clear this schedule? You cannot undo this action, but you can load your schedule again.'
-                                    )
-                                ) {
-                                    clearSchedules();
-                                    logAnalytics({
-                                        category: analyticsEnum.addedClasses.title,
-                                        action: analyticsEnum.addedClasses.actions.CLEAR_SCHEDULE,
-                                    });
-                                }
-                            }}
-                        >
-                            Clear Schedule
-                        </Button>
-                    </div>
-                </div>
-                {this.state.courses.map((course) => {
+                                    ))}
+                                    <MenuItem onClick={createCopyHandler(scheduleNames.length)}>
+                                        Copy to All Schedules
+                                    </MenuItem>
+                                </Menu>
+                            </>
+                        )}
+                    </PopupState>
+
+                    <Button
+                        sx={{ marginLeft: '4px', marginRight: '4px' }}
+                        variant="outlined"
+                        color="secondary"
+                        onClick={handleClear}
+                    >
+                        Clear Schedule
+                    </Button>
+                </Box>
+            </Box>
+
+            <Grid container spacing={2} padding={0}>
+                {courses.map((course) => {
                     return (
                         <Grid item md={12} xs={12} key={course.deptCode + course.courseNumber}>
                             <SectionTableLazyWrapper
-                                classes={this.props.classes}
                                 courseDetails={course}
                                 term={course.term}
-                                allowHighlight={false}
+                                highlightAdded={false}
                                 analyticsCategory={analyticsEnum.addedClasses.title}
-                                scheduleNames={this.state.scheduleNames}
+                                scheduleNames={scheduleNames}
                             />
                         </Grid>
                     );
                 })}
-                {this.state.customEvents.length > 0 && <Typography variant="h6">Custom Events</Typography>}
-                {this.state.customEvents.map((customEvent) => {
+
+                {customEvents.length > 0 && <Typography variant="h6">Custom Events</Typography>}
```

Comment:
Styling is off here:

<img width="636" alt="Screenshot 2023-09-06 at 3 04 36 AM" src="https://github.com/icssc/AntAlmanac/assets/100006999/5b792307-06af-4560-97c3-b1939f022f13">